test/SingleFileTestApp/SingleFileTestApp.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsTestProject>false</IsTestProject>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Let's also test on .NET Core 3.1 - as it will extract its files to disk and require a slightly different code path.

For the runtime identifier, let's choose it based on the OS and architecture the test is running on.  That way we'll get CI to test on Windows, Linux, and MacOS.

You can use MSBuild conditions in the same way we do for Sentry CLI in the Sentry.csproj file.  (Unless there's some magic to get the default RuntimeIdentifier automatically.)